lib/cartopy/crs.py

```diff
@@ -1039,12 +1039,24 @@ def y_limits(self):
 class Stereographic(Projection):
     def __init__(self, central_latitude=0.0, central_longitude=0.0,
                  false_easting=0.0, false_northing=0.0,
-                 true_scale_latitude=None, globe=None):
+                 true_scale_latitude=None, 
+                 scale_factor=None, # equivalent to 1.0
```

Comment:
Info like this comment should be in the docstring (though admittedly, there isn't one.)